### PR TITLE
docs: remove future google sign img

### PR DESCRIPTION
**Pull request by OhIAmFine**
remove img at #439

### CODE REVIEW

File: translations/README-cn.md [hunk 1]
```diff
@@ -135,8 +135,6 @@
 
 把一个（或两个）印有“[future Googler](https://github.com/jwasham/coding-interview-university/blob/master/extras/future-googler.pdf)”的图案打印出来，并用你誓要成功的眼神盯着它。
```
Comment (jwasham) [inline translations/README-cn.md]: This has it, too.

**Comment by OhIAmFine** [comment]
@jwasham  ok, I have done it.

File: translations/README-cn.md [hunk 2]
```diff
@@ -133,9 +133,7 @@
 
 ## 拥有一名 Googler 的心态
 
-把一个（或两个）印有“[future Googler](https://github.com/jwasham/coding-interview-university/blob/master/extras/future-googler.pdf)”的图案打印出来，并用你誓要成功的眼神盯着它。
-
-[![future Googler sign](https://dng5l3qzreal6.cloudfront.net/2016/Oct/Screen_Shot_2016_10_04_at_10_13_24_AM-1475601104364.png)](https://github.com/jwasham/coding-interview-university/blob/master/extras/future-googler.pdf)
+把一个（或两个）印有 “future Googler” 的图案打印出来，并用你誓要成功的眼神盯着它。
```
Comment (jwasham) [inline translations/README-cn.md]: This line can be removed.

Comment (OhIAmFine) [reply translations/README-cn.md]: > This line can be removed.

No problem.

**Comment by jwasham** [comment]
Thanks!

**Pull request merged**